lib/db.js

```diff
@@ -1056,6 +1056,9 @@ module.exports = (
 
   SAFE_URLS.createUnblockCode = new SafeUrl('/account/:uid/unblock/:unblock', 'db.createUnblockCode')
   DB.prototype.createUnblockCode = function (uid) {
+    if (! UnblockCode) {
+      throw new Error('Unblock has not been configured')
```

Comment:
I know this condition was throwing already, but now that we've made it explicit would it be more proper for it to be a rejected promise? (or changing the whole func to `async` so that the `throw` becomes a rejected promise?)

In a different repo I'd suggest we should add a test too, but the organisation of the tests here makes it kind of painful to add an extra suite just to not pass in `UnblockCode`. And as it was already failing for this case anyway, meh I guess.